auv_sim/auv_sim_bringup/launch/inc/start_bridge.launch

```diff
@@ -18,17 +18,21 @@
     </node>
   </group>
 
-  <!-- <include file="$(find auv_control)/launch/start.launch">
+  
+  <!--
+  <include file="$(find auv_control)/launch/start.launch">
```

Comment:
This is now moved to `start_navigation.launch` right? So delete the commented lines?